docs/products/valkey/howto/migrate-redis-aiven-via-console.md

```diff
@@ -102,39 +108,49 @@ While the migration is in progress, you can
 -   Stop the migration by clicking **Stop migration**. Data that has already been
     migrated is retained, and you can initiate a new migration at any time
 
-:::note
+:::note[Stopping and starting over]
 If you choose to stop the migration, this action immediately halts
 the replication of your data. However, any data that has already been
 migrated to Aiven is retained. You can initiate a new migration
 later, and this process overwrites any previously migrated
 databases.
 :::
 
-:::note[Migration attempt failed?]
-If you receive such a notification, it is important to investigate the
-possible causes of the failure and address the issues. Once you have
-resolved the underlying problems, you can initiate the migration by
-choosing **Start over**.
+:::warning[Migration attempt failed]
+If so, investigate possible causes of the failure and address the identified issues.
+Once you have resolved the underlying problems, initiate the migration by clicking
+**Start over**.
 :::
 
 ### Step 4: Close
 
-When the wizard informs you about the completion of the migration, you
-can choose one of the following options:
+When the wizard informs you about the completion of the migration, you can choose one of
+the following options:
 
--   Click **Close connection** to disconnect the databases and stop the
-    replication process if it is still active.
+-   Click **Close connection** to disconnect the databases and stop the replication process
+    if it is still active.
 
--   Click **Keep replicating** if the replication is ongoing and you
-    wish to maintain the connection open for continuous data
-    synchronization.
+-   Click **Keep replicating** to keep the connection open and the replication mode active
+    for continuous synchronization of any subsequent additions to the connected databases.
 
     ![Close database connection](/images/content/products/caching/redis-migration-complete.png)
 
-:::note[Replication mode active?]
-Your data has been successfully migrated to the designated Aiven for
-Caching database, and any subsequent additions to the connected databases
-are being continuously synchronized.
+:::note[Verify sync and plan cutover]
+
+-   **When is the replica caught up?** The Valkey replica exposes `master_sync_in_progress`.
```

Comment:
Suggestion: Update the heading from **When is the replica caught up?** to **Check initial sync status** or **Initial sync status**   Google style favors direct, task-based labels instead of question-style phrasing.